lte/gateway/python/magma/subscriberdb/rpc_servicer.py

```diff
@@ -28,6 +30,13 @@
 )
 
 enable_sentry_wrapper = get_sentry_status("subscriberdb") == SentryStatus.SEND_SELECTED_ERRORS
+suci_profile_data = NamedTuple(
```

Comment:
this was already defined on `lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py`?

Reply:
yes this is needed here, removed the same